knowage-core/src/main/java/it/eng/spagobi/commons/services/KnowageCaptchaServlet.java

```diff
@@ -64,4 +51,14 @@ private void writeImage(OutputStream outputstream, BufferedImage bufferedimage)
 		}
 	}
 
+	private int getWidthParameter(HttpServletRequest request) {
+		String widthParam = request.getParameter("width");
+		return widthParam != null ? Integer.parseInt(widthParam) : 200;
+	}
+
+	private int getHeightParameter(HttpServletRequest request) {
+		String heightParam = request.getParameter("height");
+		return heightParam != null ? Integer.parseInt(heightParam) : 50;
+	}
+
```

Comment:
This is a change in logic: width and height were retrieved from servlet config, not from request

Reply:
Fixed